lesson1/serhii_khrychov_hometask.rb

```diff
@@ -0,0 +1,24 @@
+class Foo
+  def initialize(uniq = :not_uniq)
+    @@instance_num ||= 0
+    @@instance_num += 1
+    @instance_num = @@instance_num
```

Comment:
Don't you think that names `@instance_num` and `@@instance_num` are really confusing, do you?